.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea
+target
+kickstarter.iml
+pom.xml
```

Comment:
pom.xml should not be ignored, poor Eclipse users as well as happy IDEA users use that to create IDE-specific project files